app_dart/lib/src/request_handlers/get_benchmarks.dart

```diff
@@ -30,26 +33,102 @@ class GetBenchmarks extends RequestHandler<Body> {
 
   @override
   Future<Body> get() async {
-    final String branch = request.uri.queryParameters[branchParam] ?? 'master';
-    const int maxRecords = 50;
+    const String master = 'master';
```

Comment:
Should this be a module or class level constant?

Reply:
It is only used in function level.

Reply:
This looks like a constant, how is this different from the one you defined in cocoon config?

Having the same value defined in several places makes an update of constant error prone.

Reply:
can we use a better name? from the name we don't know what it refers to.

Reply:
Replace with `defaultBranch`, and move it to cocoon config.